examples/tutorials/self-paced-training/part-4_advanced_federated_learning/chapter-10_federated_XGBoost/10.1_fed_xgboost/fed_xgboost.ipynb

```diff
@@ -0,0 +1,317 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d14f9dec-9990-46a8-b4ce-2ff81275454e",
+   "metadata": {},
+   "source": [
+    "# Federated XGBoost\n",
+    "Several mechanisms have been proposed for training an XGBoost model in a federated learning setting.\n",
+    "In these examples, we illustrate the use of NVFlare to carry out *horizontal* federated learning using two approaches: histogram-based collaboration and tree-based collaboration.\n",
+    "And *vertical* federated learning using histogram-based collaboration."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2fce92a0-79a3-4cf2-adee-bb9c4125613c",
+   "metadata": {},
+   "source": [
+    "## Horizontal Federated XGBoost\n",
+    "Under horizontal setting, each participant joining the federated learning will have part of \n",
+    "the whole data samples / instances / records, while each sample has all the features.\n",
+    "\n",
+    "### Histogram-based Collaboration\n",
+    "The histogram-based collaboration federated XGBoost approach leverages NVFlare integration of [federated learning support](https://github.com/dmlc/xgboost/issues/7778) in the XGBoost open-source library,\n",
+    "which allows the existing *distributed* XGBoost training algorithm to operate in a federated manner,\n",
+    "with the federated clients acting as the distinct workers in the distributed XGBoost algorithm.\n",
+    "\n",
+    "In distributed XGBoost, the individual workers share and aggregate gradient information about their respective portions of the training data,\n",
```

Comment:
the individual workers -> "individual workers"

Reply:
updated